core/blockchain_optimism.go

```diff
@@ -0,0 +1,27 @@
+package core
+
+import (
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/metrics"
+)
+
+// OPStack additions
+var (
+	headBaseFeeGauge     = metrics.NewRegisteredGauge("chain/head/basefee", nil)
+	headGasUsedGauge     = metrics.NewRegisteredGauge("chain/head/gas_used", nil)
+	headBlobGasUsedGauge = metrics.NewRegisteredGauge("chain/head/blob_gas_used", nil)
+
+	headGasUsedHist     = metrics.NewRegisteredHistogram("chain/head/gas_used_hist", nil, metrics.NewExpDecaySample(1028, 0.015))
+	headBlobGasUsedHist = metrics.NewRegisteredHistogram("chain/head/blob_gas_used_hist", nil, metrics.NewExpDecaySample(1028, 0.015))
```

Comment:
Any reason for `1028`? If we wanted to cover e.g. 10min of L2 blocks, we could pick `10 * 30 = 300` instead.

Reply:
Yeah I think we can leave it like this for now, given that there is the other paramter which we might want to tune as well. If we roll with this, we can circle back and tweak once we try it out.